### Slot 2-1 Long Calculations

#### Task 1
Import pi from math modules

In [24]:
from math import pi

#### Task 2
Calculate power when rpm and Torque is given using this equation

`n = 1800 rpm`

`Torque = 0.8 N.m` 

$Power, P= \frac{2 \pi n}{60} \times Torque$

In [25]:
n=1800
Torque = 0.8
Power = Torque * n * 2 * pi / 60
print('Power =', Power, 'W')

Power = 150.79644737231007 W


#### Task 3
Rounding the Power to 3 decimals

Example:

`Power = round(Torque * n * 2 * pi / 60,2)`

Output:

`Power = 150.8 W`

In [26]:
Power = round(Torque * n * 2 * pi / 60,3)
print('Power =', Power, 'W')

Power = 150.796 W


#### Task 4
A solid square rod with diameter, d=5mm is pulled along the axis with an equivalent force of Force = (300 cos 15)N. Calculate the stress using this equation.

$\sigma = \dfrac{Force}{Area}$

In [27]:
from math import cos, radians

In [28]:
d = 0.005
Force = 300 * cos(radians(15))
Area = pi * d**2/4
stress = Force / Area
print('Stress =', stress, 'Pa')

Stress = 14758259.51174675 Pa


#### Task 5
Convert the Stress to MPa

In [32]:
print(f'Stress = {stress/1e6:.2f} MPa')
# or
# print('Stress = {:.2f} MPa'.format(stress/1e6))


Stress = 14.76 MPa


#### Task 6
Convert the Stress to Engineering notations

In [33]:
print(f'Stress = {stress/1e6:.2e} MPa')

Stress = 1.48e+01 MPa
